tests/conftest.py

```diff
@@ -1,28 +1,36 @@
 import os
-from typing import Generator
+from contextlib import contextmanager
+from pathlib import Path
+from typing import Iterator
 
 import pytest
 
 
+@contextmanager
+def create_tmp_file(path: Path, data: str) -> Iterator[str]:
```

Comment:
We should probably be using this https://docs.python.org/3/library/tempfile.html

Reply:
Yes good idea! Is it okay to do it in another PR, so the fix gets merged?

I can create an issue for it.

Reply:
Sure

Reply:
#159